gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -319,6 +319,18 @@ user:
       #subject: Please reset your password
     token:
     #expire-after: 300 # the end of validity of the token in seconds for reset password email (default 300 = 5 minutes)
+  activity:
```

Comment:
We may have to document these but we should not have this configuration directly in the gravitee.yml